pkg/cmd/issue/create/create.go

```diff
@@ -279,7 +292,7 @@ func createRun(opts *CreateOptions) (err error) {
 				Repo:      baseRepo,
 				State:     &tb,
 			}
-			err = prShared.MetadataSurvey(opts.Prompter, opts.IO, baseRepo, fetcher, &tb)
+			err = prShared.MetadataSurvey(opts.Prompter, opts.IO, baseRepo, fetcher, &tb, projectsV1Support)
```

Comment:
This block is untested, however I'm going to rely on the fact it's a simple value pass and the `MetadataSurvey` tests to avoid blowing up this work: https://github.com/cli/cli/pull/10815/files#diff-e76ab3c0932c8cf32002e9ad1bd0c61a3fd6f26d9bdcbbc65fdb30ec14e41a41R133-R175